README.md

```diff
@@ -69,7 +70,7 @@ cmake .
 after which you can use make to build the project
 
 ```shell
-make -j4
+make -j
```

Comment:
-j uses as many threads as it wants. I don't think we should do that. Lets just make it `make`

Reply:
Ok. I in my experience "-j" seems like the way to go, but I guess we don't know what hardware they have. I've made not change. As well as a similar change to make instructions for building xgboost.